rotkehlchen/chain/evm/decoding/curve_lend/constants.py

```diff
@@ -0,0 +1,3 @@
+from typing import Final
+
+CURVE_VAULT_API: Final = 'https://api.curve.fi/v1/getLendingVaults/all'
```

Comment:
Why create an entire file for something that's only used in one place. Please do not do that.